### DISCUSSION TITLE
SplitButton: Add loading control

### DISCUSSION THREAD

**Discussion by byronogis** [primefaces/community#2770] [PrimeVue]
See https://github.com/primefaces/primevue/issues/4221

**Comment by j0rgedev** [comment]
Supporting this request. However, it wouldn't be that hard to implement it by your own

**Reply by cmcnicholas** [reply]
> Supporting this request. However, it wouldn't be that hard to implement it by your own

this is pretty difficult, you can abuse the buttonProps but effectively you need to reimplement a good amount of this component by defining the main and dropdown button again, at which point you've just wrote something separate.
